libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -271,6 +278,31 @@ bool AC_PrecLand::get_target_velocity_relative_cms(Vector2f& ret)
     return true;
 }
 
+// get the absolute velocity of the vehicle
+void AC_PrecLand::get_target_velocity_cms(const Vector2f& vehicle_velocity_cms, Vector2f& target_vel_cms)
+{
+    if (!_moving) {
+        // the target should not be moving
+        target_vel_cms.zero();
+        return;
+    }
+    if ((EstimatorType)_estimator_type.get() == EstimatorType::RAW_SENSOR) {
```

Comment:
Should use `AP_Enum<EstimatorType>` instead of `AP_Int8` as a type for `_estimator_type`

Pre-existing